save-agent/src/linuxX64Test/kotlin/com/saveourtool/save/agent/SaveAgentTest.kt

```diff
@@ -22,13 +22,6 @@ import kotlinx.coroutines.runBlocking
 import kotlinx.serialization.PolymorphicSerializer
 
 class SaveAgentTest {
-    init {
```

Comment:
Order of initialization is essential for this test to work, unfortunately. That's why CI is failing.
Because of poor API of `kotlin-test`, there is no other way to call a `BeforeAll` kind of logic.